cmd/etcd.go

```diff
@@ -180,6 +185,20 @@ func RestoreEtcdSnapshot(
 	return APIURL, caCrt, clientCert, clientKey, certs, err
 }
 
+func validateETCDCerts(etcdHosts []*hosts.Host, state cluster.State) error {
+	if state.RancherKubernetesEngineConfig == nil {
+		// possibly already started a restore
+		return nil
+	}
+	for _, etcdHost := range etcdHosts {
+		certName := pki.GetCrtNameForHost(etcdHost, pki.EtcdCertName)
```

Comment:
I am not sure that we need actually to check for the etcd cert, since it should be regenerated as a part of rke updates which will be executed as a part of the restore, meanwhile I think we should check for other certs